### ISSUE TITLE
Limiter le nombre d'éléments visibles dans le composant Autocomplete

### ISSUE THREAD

**Issue by patryyyck** [SDV-Plurimedia/ngx-widgets#27]
Bonjour,

Serait-il possible d'avoir une "propriété" permettant de limiter le nombre de lignes visibles dans le cas où il y en aurait vraiment beaucoup dans le composant Autocomplete?

Deux propositions:
- Ajouter un paramètre nombre de ligne au niveau du composant
- Modifier le style de ce dernier en mettant en paramètre le max-height

Merci d'avance

**Comment by TwanoO67** [comment]
Hello @patryyyck ,

Pour le moment c'est faisable uniquement avec le customSearch (en proposant ta propre maniere de filtre, celle-ci pouvant etre limité en nombre de resultat)

Mais cette option serait une bonne idée, la problématique serait surtout de gérer la "pagination".
A savoir, si le résultat recherché n'est pas dans la liste limité, comment fais-t-on pour accéder à la suite

**Comment by patryyyck** [comment]
Effectivement, filtrer les résultats peut être une bonne solution à condition de trouver un moyen de gérer la pagination.
Mais dans un premier temps, sans toucher au nombre de lignes retournées actuellement, le but serait  "juste" de limiter le nombre de lignes visibles (quitte à avoir une scrollbar conséquente).
